test/Account/unlock.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as sinon from 'sinon';
+import { assert } from 'chai';
+
+import Account from '../../src/Account';
+import SpyAssert from '../utils/SpyAssert';
+
+const Web3 = require('web3');
+
+describe('Unlock', () => {
+  let web3: any;
+  let validPassword: string;
+  let accountCreationResponse: Account;
+  let accountObject: Account;
+
+  beforeEach(() => {
+    web3 = new Web3();
+    validPassword = 'validPassword';
+    accountCreationResponse = Account.create(web3, validPassword);
+    accountObject = new Account(
+      accountCreationResponse.address,
+      accountCreationResponse.encryptedKeyStore
+    );
+  });
+
+  it('should unlock successfully with valid password', () => {
+    const dummyAccount = web3.eth.accounts.create();
```

Comment:
```suggestion
    const dummyAccount = await web3.eth.accounts.create();
```

Reply:
This is a sync command as per https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html.